tools/crm_mon_print.c

```diff
@@ -1137,9 +1137,9 @@ print_status(pcmk__output_t *out, mon_output_format_t output_format,
             out->info(out, "%s", "");
         }
 
-        if (is_set(show, mon_show_fencing)) {
+        if (is_set(show, mon_show_fence_worked)) {
             print_stonith_history(out, stonith_history, mon_ops);
-        } else {
+        } else if (is_set(show, mon_show_fence_pending)) {
             print_stonith_pending(out, stonith_history, mon_ops);
```

Comment:
I think we'll need a bigger refactor of this part; print_stonith_history() will print out pending and succeeded. I'm thinking we can combine print_stonith_history() and print_stonith_pending(), and use mon_ops to tell what header to print etc.

Reply:
This could potentially also be merged with print_stonith_history_full.

Reply:
On second thought, it's simpler to just leave print_stonith_history_full alone.  There's several other similar functions that could perhaps all be smashed into one, but it'd make for some overly complicated conditionals.